### ISSUE TITLE
ERROR for new release 3.0.0

### ISSUE THREAD

**Issue by georgeyean** [quanteda/quanteda.sentiment#13]
Hi,

It seems the latest release has fundamental issues? it was working fine like two months ago:

```
valence(data_dictionary_LSD2015) <- list(positive = 1, 
                                          negative = -1, 
                                         neg_negative = 1, 
                                           neg_positive = -1)

```

Error in UseMethod("valence<-") : 
  no applicable method for 'valence<-' applied to an object of class "dictionary3"

Thanks.

**Comment by ajgoecke** [comment]
Hi, I get the exactly same error since today and everything worked perfectly fine until yesterday evening... I have to submit my project for university next week and it totally relies on the quanteda.sentiment valence function... What could I do now?

Best wishes, 
Anna

**Comment by kbenoit** [comment]
This seems to be working fine:
``` r
library("quanteda")
## Package version: 3.2.0.9000
## Unicode version: 13.0
## ICU version: 69.1
## Parallel computing: 8 of 8 threads used.
## See https://quanteda.io for tutorials and examples.
library("quanteda.sentiment")
## 
## Attaching package: 'quanteda.sentiment'
## The following object is masked from 'package:quanteda':
## 
##     data_dictionary_LSD2015

valence(data_dictionary_LSD2015) <- c(
  positive = 1,
  negative = -1,
  neg_negative = 1,
  neg_positive = -1
)

textstat_valence(data_corpus_inaugural[1:2], data_dictionary_LSD2015)
##            doc_id sentiment
## 1 1789-Washington 0.4787879
## 2 1793-Washington 0.5384615

# equivalent (and faster)
textstat_polarity(data_corpus_inaugural[1:2], data_dictionary_LSD2015,
  fun = sent_relpropdiff
)
##            doc_id sentiment
## 1 1789-Washington 0.4787879
## 2 1793-Washington 0.5384615
```

@ajgoecke can you provide the output from `reprex::reprex()`?

**Comment by maxsowi** [comment]
Does anyone have a solution for this error? It occurred to me now as well while the script was running perfectly fine before.  

> data_dictionary_LSD2015_pos_neg <- data_dictionary_LSD2015[1:2]
Error in UseMethod("valence") : 
  no applicable method for 'valence' applied to an object of class "dictionary3"

Many thanks in advance!

**Comment by kbenoit** [comment]
This should be fixed in v0.3 now, please try.
